src/frontend/src/service/currency-format.ts

```diff
@@ -0,0 +1,8 @@
+export default function currencyFormat(input: number) {
+  var wholePart = Math.trunc(input);
+  var wholeStr = wholePart.toLocaleString();
+  var fractionPart = Math.abs(input) % 1;
+  var fractionStr = fractionPart.toFixed(2).substring(1);
+  var rt = '$' + wholeStr + fractionStr;
```

Comment:
`const`s here too , also I suggest renaming the rt variable to `currencyString`

Reply:
Done